azure-pipelines.sds.pr-release.yml

```diff
@@ -5,7 +5,7 @@ resources:
     - repository: azTemplates
       type: github
       name: hmcts/azure-devops-templates
-      ref: master
+      ref: fix-vh-templates-for-dotnet-8
```

Comment:
will we update this to master later?

Reply:
yup! might create a second or once all of the branches have been updated